contracts/coordinator/src/msg.rs

```diff
@@ -41,6 +46,55 @@ pub enum ExecuteMsg {
     },
     #[permission(Specific(prover))]
     SetActiveVerifiers { verifiers: HashSet<String> },
+
+    #[permission(Any)]
+    InstantiateChainContracts {
+        deployment_name: String,
+        params: Box<DeploymentParams>,
+    },
+}
+
+#[cw_serde]
+pub enum DeploymentParams {
+    Manual {
```

Comment:
what does `Manual` mean in this context? could you maybe add a comment explaining why this is an enum and the meaning of manual?